dev/tasks/docs/github.linux.yml

```diff
@@ -21,13 +21,59 @@
 
 jobs:
   test:
-    name: Docs Preview 
+    name: Docs Preview
     runs-on: ubuntu-latest
 {{ macros.github_set_env(env) }}
     steps:
       {{ macros.github_checkout_arrow(fetch_depth=fetch_depth|default(1))|indent }}
       {{ macros.github_install_archery()|indent }}
 
+      - name: Free up disk space

```

Comment:
Is it possible to add some kind of macro for this step instead of repeating it in two different files?

Reply:
Yes. But the debug option approach is better.